Dshop/apps/products_catalogue/templates/products_catalogue/category_detail.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<style>
+    .card-img {
+    width: 100%;
+    height: 13vw;
+}
+    .embed-responsive .card-img {
+    object-fit: cover;
+}
+
+
+</style>
+
+<div class="container px-4 py-5" id="featured-3">
+    <h2 class="pb-2 border-bottom">{{ category.name }}</h2>
+    <div class="row g-4 py-5 row-cols-1 row-cols-lg-3">
+        {% for product in category.product_set.all %}
+        {% if product.is_active %}
```

Comment:
This if should not be needed, as there should be no inactive products here. 
What I would do, is I'd add a method on category model to get active products (return queryset), and then use 
```
{% for product in category.active_products.all %}
```
This way you get rid of this IF and you can use the above `category.active_products` in several places - ie. when filtering by price or search.

Reply:
done!